blocks/metrics/prometheus.go

```diff
@@ -0,0 +1,50 @@
+package metrics
+
+import (
+	"github.com/spacemeshos/go-spacemesh/metrics"
+)
+
+const (
+	subsystem     = "blocks"
+	diffTypeLabel = "diff_type"
+)
+
+// AvgLayerBlockSize checks average block size.
+var AvgLayerBlockSize = metrics.NewHistogram(
+	"avg_layer_block_size",
```

Comment:
i think avg_ prefix needs to be removed from metrics. histogram is not about averages, avg prefix will only confuse